exercises/prime-factors/src/test/java/PrimeFactorsCalculatorTest.java

```diff
@@ -8,20 +7,22 @@
 import java.util.Collection;
 import java.util.List;
 
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
 import static org.junit.Assert.assertEquals;
 
 @RunWith(Parameterized.class)
-public class PrimeFactorsTest {
+public class PrimeFactorsCalculatorTest {
 
     private long input;
     private List<Long> expectedOutput;
 
     @Parameters(name="Prime factors of {0} = {1}")
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][]{
-                {1L, Arrays.asList()},
-                {2L, Arrays.asList(2L)},
-                {3L, Arrays.asList(3L)},
+                {1L, emptyList()},
+                {2L, singletonList(2L)},
+                {3L, singletonList(3L)},
```

Comment:
I'm more used to seeing this as `Collections.emptyList()` and `Collections.singletonList()` without the static import. That might make it more clear to someone who hasn't encountered them before?